_pyansys-dashboard-vulnerabilities-upload/action.yml

```diff
@@ -0,0 +1,192 @@
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+name: >
+  Upload vulnerability reports to PyAnsys Dashboard database.
+
+description: >
+  This action is only intended to be used by repositories that are part of the
+  PyAnsys Dashboard project. It uploads the vulnerability reports generated by
+  the `safety <https://pyup.io/safety/>`_  and `bandit <https://bandit.readthedocs.io/en/latest/>`_
+  tools to the PyAnsys Dashboard database.
+
+inputs:
+
+  # Required inputs
+  pyansys-dashboard-token:
+    description: >
+      Token used to clone the PyAnsys Dashboard repository and upload the
+      vulnerability reports.
+    required: true
+    type: string
+
+  pyansys-dashboard-credentials:
+    description: >
+      Credentials to access the PyAnsys Dashboard database. They
+      are available as a secret inside the ``ansys`` and ``ansys-internal``
+      organizations.
+    required: true
+    type: string
+
+  dry-run:
+    description: >
+      Whether to run or not this action in testing mode. Testing mode executes
+      all the action steps except the releasing artifacts one. This allows
+      developers to verify the health of the action once integrated in their
+      CI/CD pipelines without actually publishing artifacts.
+    required: true
+    type: boolean
+
+  repository:
+    description: >
+      Full name of the repository for which the vulnerability reports are generated.
+    required: true
+    type: string
+
+  safety-report:
+    description: >
+      Path to the safety report file.
+    required: true
+    type: string
+
+  bandit-report:
+    description: >
+      Path to the bandit report file.
+    required: true
+    type: string
+
+  hide-log:
+    description: >
+      Whether to hide the output log of the GitHub action.
+      By default, the log is available to anyone for public repositories.
+      This would disclose any potential vulnerabilities to anyone.
+    required: true
+    type: boolean
```

Comment:
Can we add a default value and set it to `true`?

Reply:
This is a private action. It is only called from the ansys/check-vulnerabilities action. We are passing the value used in that action here.

Following the same approach we have with other private actions, all inputs are compulsory and none have a default value.

Reply:
I need to change the description though - that's my bad for copypasting